slices/grep.yaml

```diff
@@ -0,0 +1,20 @@
+package: grep
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre3_libs
+    contents:
+      /bin/grep:
+
+  deprecated:
+    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
+    essential:
+      - bash_bins
+      - libpcre3_libs
+    contents:
+      /bin/egrep:
+      /bin/fgrep:
+      /usr/bin/rgrep:
```

Comment:
Some of the paths are wrong. See here: https://packages.ubuntu.com/noble/amd64/grep/filelist.

```
/usr/bin/egrep
/usr/bin/fgrep
/usr/bin/rgrep
```

Reply:
Oh ho, good catch - I only have a Jammy machine handy.